homeworks/misha.kolosov_mwkolosov/homework_7/main.test.js

```diff
@@ -0,0 +1,24 @@
+const { fib } = require('./main.js');
+
+describe('Fibonacci', () => {
+    describe('Сheck value', () => {
+        test('First value', () => {
```

Comment:
tests should have informative description like 'first element of returned array should be 0'